subsystems/viamserver/viamserver.go

```diff
@@ -137,7 +137,7 @@ func (s *viamServer) Start(ctx context.Context) error {
 		if s.cmd.ProcessState != nil {
 			s.lastExit = s.cmd.ProcessState.ExitCode()
 			if s.lastExit != 0 {
-				s.logger.Errorw("non-zero exit code", "exit code", s.lastExit)
+				s.logger.Error("non-zero exit code: %d", s.lastExit)
```

Comment:
Needs to be Errorf() not Error() (will correct after 108 is in and can more easily rebase)